gamemode/modules/util/cl_model_drawing.lua

```diff
@@ -0,0 +1,51 @@
+local viewSize = 1024
+
+RVR.Util = RVR.Util or {}
+
+local mats = {}
+function RVR.Util.getModelTexture( model, pos, ang )
+    if mats[model] then return mats[model] end
+    pos = pos or Vector( 0, 0, 0 )
+    ang = ang or Angle()
+
+    local texture = GetRenderTargetEx( "rvr_model_" .. model, viewSize, viewSize, RT_SIZE_DEFAULT, MATERIAL_RT_DEPTH_SHARED, 0, CREATERENDERTARGETFLAGS_UNFILTERABLE_OK, IMAGE_FORMAT_RGBA8888 )
```

Comment:
Could do with some new lines
Whoever wrote this must be insane or something

Reply:
wow... you wrote this

Reply:
dunno what you're talking about